src/components/Navbar/index.jsx

```diff
@@ -115,14 +118,15 @@ const Navbar = ({ onSettings }) => {
               className={'nav-text'}
               style={{ textDecoration: 'none' }}
             >
-              Avocadoro
+              Avocadoro<span> </span>
```

Comment:
What's this?

Reply:
This is to introduce a space between the word `Avocadoro` and the actual count.